spec/src/modules/browse.js

```diff
@@ -607,6 +626,24 @@ describe('ConstructorIO - Browse', () => {
       });
     });
 
+    it('Should return a response with valid ids and hiddenFields', (done) => {
+      const hiddenFields = ['hiddenField1', 'hiddenField2'];
+      const { browse } = new ConstructorIO({
+        apiKey: testApiKey,
+        fetch: fetchSpy,
+      });
+
+      browse.getBrowseResultsByItemIds(ids, { hiddenFields }).then((res) => {
+        const requestedUrlParams = helpers.extractUrlParamsFromFetch(fetchSpy);
+
+        expect(res).to.have.property('request').to.be.an('object');
+        expect(res).to.have.property('response').to.be.an('object');
+        expect(res).to.have.property('result_id').to.be.an('string');
+        expect(requestedUrlParams).to.have.property('hidden_fields').to.eql(hiddenFields);
```

Comment:
Should we have a

```expect(res.request.hidden_fields).to.eql(hiddenFields);```

check here as well?

Reply:
Yes. Thanks for catching it!